addons/im_livechat/report/im_livechat_report_channel.py

```diff
@@ -82,7 +83,8 @@ def _cte(self) -> SQL:
             channel_member_history AS (
                 SELECT channel_id,
                        BOOL_OR(livechat_member_type = 'agent') as has_agent,
-                       BOOL_OR(livechat_member_type = 'bot') as has_bot
+                       BOOL_OR(livechat_member_type = 'bot') as has_bot,
+                       MIN(CASE WHEN livechat_member_type = 'visitor' AND partner_id IS NOT NULL THEN partner_id END) AS visitor_partner_id
```

Comment:
`MIN` skips null values sor the `AND` only complexify the line, we can remove it.